examples/cfq/utils.py

```diff
@@ -0,0 +1,234 @@
+# Copyright 2020 The Flax Authors.
```

Comment:
This file seems to have utils specific to input_pipeline.py. I suggest to either move these util functions into input_pipeline.py, or otherwise rename this file to input_pipeline_utils.py. "utils" sounds very generic.